nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/admin/admin-ai-export.html

```diff
@@ -0,0 +1,169 @@
+<dom-module id="nuxeo-admin-ai-export">
+    <template>
+        <style>
+            #statsResult {
+                height: 60vh;
+            }
+
+            .row-container {
+                @apply --layout-horizontal;
+                @apply --layout-wrap;
+            }
+        </style>
+
+        <nuxeo-operation id="aiStats" op="AI.DatasetStats" response="{{stats}}"></nuxeo-operation>
+        <nuxeo-operation id="aiExport" op="AI.DatasetExport" ></nuxeo-operation>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <paper-textarea id="queryInput"
+                            type="search"
+                            label="[[i18n('NXQL Query')]]"
+                            value="{{query}}"
+                            placeholder="[[i18n('imaging.query.placeholder')]]"
+                            autofocus>
+            </paper-textarea>
+            <paper-input id="inputs" label="[[i18n('admin.ai.inputs')]]" value="{{in_props}}"></paper-input>
+            <paper-input id="outputs" label="[[i18n('admin.ai.outputs')]]" value="{{out_props}}"></paper-input>
+            <paper-input id="split" label="[[i18n('admin.ai.split')]]" value="{{split_prop}}"></paper-input>
+            <paper-button on-tap="_recompute">[[i18n('admin.ai.stats.action')]]</paper-button>
+            <paper-button on-tap="_export">[[i18n('admin.ai.export.action')]]</paper-button>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.stats')]]">
+            <div class="row-container">
+                Total possible: [[_total.numericValue]] &nbsp;
+                Valid count: [[_count.numericValue]]
+            </div>
+            <nuxeo-data-table id="statsResult" items="[[_filter(stats)]]">
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.field')]]" flex="10">
+                    <template>
+                        [[item.field]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.type')]]" flex="10">
+                    <template>
+                        [[item.type]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.value')]]" flex="50">
+                    <template>
+                        [[_displayVal(item)]]
+                    </template>
+                </nuxeo-data-table-column>
+            </nuxeo-data-table>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <div class="row-container">
+               [[export]]
+            </div>
+        </nuxeo-card>
+    </template>
+    <script>
+        Polymer({
+            is: 'nuxeo-admin-ai-export',
+            behaviors: [Nuxeo.LayoutBehavior],
+            properties: {
+                in_props: {
```

Comment:
Avoid using naming public element properties with underscores, since they will fail to convert to html attributes. See [Property name to attribute name mapping](https://polymer-library.polymer-project.org/2.0/docs/devguide/properties#property-name-mapping) for more.